src/core/sys/posix/sys/types.d

```diff
@@ -852,7 +852,7 @@ else version (Darwin)
 {
     version (D_LP64)
     {
-        enum __PTHREAD_SIZE__               = 1168;
```

Comment:
This change applies to existing prorgams. Correct me if I'm wrong but these new values appear to be the defaults for linux, ... which differ from OSX IIRC.

Reply:
I'm not sure I understand. The new values are what's written in the header files in the SDK for macOS, iOS, tvOS and watchOS. I don't know if this has been updated or where the original values come from.

Reply:
You're right, seems like the current values are outdated (they originate from the first official commit to druntime).

E.g. [this outdated FAQ](https://developer.apple.com/library/archive/qa/qa1419/_index.html) mentions 512KB as the default stack size for secondary threads in pthread.